### PR TITLE
Translation for _toctree, U0, U1, U2, U3 to zh-CN

### PR DESCRIPTION
**Pull request by fisheggg**
#59

Hi audio team! Thanks for the amazing contents. 
I just finished the first part of translation to zh-CN, will hopefully continue working on the coming units. 

Cheers! 🍻 
Arthur

### CODE REVIEW

**Comment by fisheggg** [comment]
> Thank you for all of your work translating the course! There are a couple of changes missing. To build the pages, you need to modify `.github/workflows/build_documentation.yml` and `.github/workflows/build_pr_documentation.yml` to include the new language. You can see an example in this PR #68 Fill free to ping me for a new review, once you've added these changes

Thanks for the tips @MKhalusova ! I've just added the language to the files. Please take a look when you have time 😇 
Will keep working on the new chapters!

Best,
Arthur

File: chapters/zh-CN/_toctree.yml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    title: Seq2Seq结构
+  - local: chapter3/classification
+    title: 音频分类结构
+  - local: chapter3/quiz
+    title: 习题
+    quiz: 3
+  - local: chapter3/supplemental_reading
+    title: 补充阅读
+
+- title: 第4单元：构建音乐风格分类器
```
Comment (MKhalusova) [inline chapters/zh-CN/_toctree.yml:55]: Please comment Chapter 4 in the TOC. As this chapter has not been translated yet, and there are no corresponding files, this breaks the doc-build.

**Review by MKhalusova — CHANGES_REQUESTED** [review]
Thank you for iterating on the PR! Please make sure all the checks run green, then, we'll be able to merge.

File: chapters/zh-CN/_toctree.yml (hunk 1 above)
Comment (fisheggg) [reply chapters/zh-CN/_toctree.yml]: Commented U4 and events part in TOC!

**Review by MKhalusova — APPROVED** [review]
Thank you for iterating over the changes! Looks good to me.

**Pull request merged**